Source/Eto/Forms/Controls/Splitter.cs

```diff
@@ -87,11 +89,40 @@ protected virtual void OnPositionChanged(EventArgs e)
 
 		#endregion
 
+		/// <summary>
+		/// Initializes a new instance of the <see cref="T:Eto.Forms.Splitter"/> class.
+		/// </summary>
+		public Splitter()
+		{
+			panel1MinimumSize = 0;
+			panel2MinimumSize = 0;
```

Comment:
I'll do some testing on the other platforms before merging this in.  Thanks again for the PR!